Java/tests/src/test/java/com/microsoft/recognizers/text/tests/sequence/SequenceTest.java

```diff
@@ -35,7 +36,25 @@ protected void recognizeAndAssert(TestCase currentCase) {
         List<ModelResult> results = recognize(currentCase);
 
         // assert
-        assertResultsWithKeys(currentCase, results, Arrays.asList(ResolutionKey.Value));
+        assertResults(currentCase, results, getKeysToTest());
+    }
+
+    @Override
+    protected void assertModel(ModelResult expected, ModelResult actual, TestCase currentCase, List<String> testResolutionKeys) {
+        super.assertModel(expected, actual, currentCase, testResolutionKeys);
+
+        // Number and NumberWithUnit are supported currently.
```

Comment:
Is this ok? Also, this comment makes me think that we shouldn't add the validation of `start` and `end`

Reply:
Since neither JS nor Python are validating these properties, I say we remove these validations

Reply:
Before removing, C# does this validation? If so, why Java is broken? Maybe we can improve the parity for the fix.